Makefile

```diff
@@ -7,8 +7,8 @@ SHELL := bash
 .ONESHELL:
 
 CERT_MANAGER_VERSION ?= 1.6.2
-BUNDLE_VERSION ?= ${CERT_MANAGER_VERSION}-rc1
-BUNDLE_CHANNELS ?= candidate
+export BUNDLE_VERSION ?= ${CERT_MANAGER_VERSION}
+BUNDLE_CHANNELS ?= candidate stable
```

Comment:
I've documented that final releases should be added to both the candidate and stable channels while release candidates should only be added to the candidate channel.